AppCenter/AppCenterTests/MSACChannelGroupDefaultTests.m

```diff
@@ -229,6 +229,40 @@ - (void)testPause {
   OCMVerify([channelMock pauseWithIdentifyingObject:identifyingObject]);
 }
 
+- (void)testResumeWithConcurrentChannelsModification {
+  // If
```

Comment:
According to our styling, there is always an empty line before comments. Please add to other places too.